.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Issue tracker is **ONLY** used for reporting bugs on documents (PEPs), not
+bugs in Python.
```

Comment:
We could link to https://devguide.python.org/tracker/ as the place to report bugs in Python (and let's call it CPython).